server/odc-service/src/main/java/com/oceanbase/odc/service/datatransfer/DataTransferService.java

```diff
@@ -166,76 +158,56 @@ public void init() {
             Long connectionId = transferConfig.getConnectionId();
             PreConditions.validArgumentState(connectionId != null, ErrorCodes.BadArgument,
                     new Object[] {"ConnectionId can not be null"}, "ConnectionId can not be null");
             ConnectionConfig connectionConfig = connectionService.getForConnectionSkipPermissionCheck(connectionId);
             transferConfig.setConnectionInfo(connectionConfig.toConnectionInfo());
             injectSysConfig(connectionConfig, transferConfig);
 
-            boolean transferData = transferConfig.isTransferData();
-            boolean transferSchema = transferConfig.isTransferDDL();
-            if (transferType == DataTransferType.IMPORT) {
-                List<String> importFileNames = transferConfig.getImportFileName();
-                if (!transferConfig.isCompressed()) {
-                    copyImportScripts(importFileNames, transferConfig.getDataTransferFormat(), workingDir);
-                } else {
-                    copyImportZip(importFileNames, workingDir);
-                }
-                BaseParameterFactory<LoadParameter> factory =
-                        new LoadParameterFactory(workingDir, logDir, connectionConfig);
-                LoadParameter parameter = factory.generate(transferConfig);
-                ImportDataTransferTask transferTask;
-                try {
-                    transferTask = new ImportDataTransferTask(parameter, transferData, transferSchema);
-                } catch (Exception e) {
-                    logger.warn("Failed to init load task, reason : {}", e.getMessage(), e);
-                    throw e;
-                }
-                return BaseDataTransferTask.start(executor, transferTask);
-            } else if (transferType == DataTransferType.EXPORT) {
-                BaseParameterFactory<DumpParameter> factory = new DumpParameterFactory(workingDir, logDir,
-                        connectionConfig, dataTransferAdapter.getMaxDumpSizeBytes(),
-                        dataTransferProperties.getCursorFetchSize(), maskingService);
-                DumpParameter parameter = factory.generate(transferConfig);
-                ExportDataTransferTask transferTask;
-                try {
-                    transferTask = new ExportDataTransferTask(
-                            parameter, transferData, transferSchema, dataTransferAdapter);
-                } catch (Exception e) {
-                    logger.warn("Failed to init dump task, reason : {}", e.getMessage(), e);
-                    throw e;
-                }
-                transferTask.setMergeSchemaFiles(transferConfig.isMergeSchemaFiles());
-                return BaseDataTransferTask.start(executor, transferTask);
-            }
-            throw new IllegalArgumentException("Illegal transfer type " + transferType);
+            // task placeholder
+            Holder<DataTransferCallable> jobHolder = new Holder<>();
```

Comment:
don't use holder, there are two suggestions:
1. Get `DataTransferCallable` in DataTransferService directly and make it as an input for DataTransferTaskBuilder.
2. You can generate `DataTransferCallable` in DataTransferTask just like what you do currently, but do not use `Holder` to get this job, you'd better get it by getter method of `DataTransferTask`.

Reply:
done. Added a getter method.